src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Tables/Workitem.sql

```diff
@@ -26,6 +27,7 @@ CREATE UNIQUE NONCLUSTERED INDEX IX_Workitem_WorkitemUid_PartitionKey ON dbo.Wor
 )
 INCLUDE
 (
+    Watermark,
```

Comment:
We are including watermark as a filter. We should have a new index with watermark and workitemkey

Reply:
Wouldn't IX_Workitem_WorkitemUid_PartitionKey be enough?

Reply:
No, where is the watermark. It should be part of the index, else we will be doing index scan

Reply:
Updated the index scripts